README.md

```diff
@@ -93,17 +96,39 @@ channel on the [Firecracker Slack](https://tinyurl.com/firecracker-microvm).
 
 ### Building
 
-Each of the components requires Go 1.11 and utilizes Go modules. You must have a
-properly set up Go toolchain capable of building the components. Note that
-Modules are an experimental opt-in feature in Go 1.11, there are two ways to
-activate module support:
+#### Binary components ####
 
-* Invoke the `go` command in a directory outside of the `$GOPATH`.
-* set `GO111MODULE=on` before invoking the `go` command.
+**Note** The binary components require Go 1.11 or greater, and they
+utilize Go modules. You must have a properly set up Go toolchain
+capable of building the components.
 
-The devicemapper snapshotter requires `dmsetup` command line tool to be
-installed and available on your computer. On Ubuntu, it can be installed
-with `apt-get install dmsetup` command.
+Use `make` to build the binary components:
+
+    make all
+
+#### Root image ####
+
+**Note** The root image build is performed in a Docker container, and
+the tooling assumes that you have the ability to interact with Docker
+directly (i.e. without `sudo` or similar).
+
+The `image` make target can be used to invoke the root image build. As
+an optimization, you can specify a local Debian package mirror via the
+`DEBMIRROR` environment variable when generating the image, e.g.
+
+    make image DEBMIRROR=http://my-local-mirror/debian
+
+**Note** The generated filesystem image will be stored in
+`tools/image-builder/rootfs.img`. In order to use this image with
+firecracker-containerd, you must pass the following parameters to the
```

Comment:
> In order to use this image with firecracker-containerd, you must pass the following parameters to .. via the `kernel_args` setting in

would it make sense to mention that we have to copy the image to a proper location first ?

Reply:
Either needs to be copied or you need to specify the path in the runtime configuration file.

Reply:
This text is gone now, in favor [the getting started guide](docs/getting-started.md), which already documented copying the image to the final location.